torch_xla/csrc/tensor_methods.cpp

```diff
@@ -953,6 +953,16 @@ XLATensor XLATensor::eye(xla::int64 lines, xla::int64 cols,
       device, element_type);
 }
 
+void XLATensor::eye_out(XLATensor& out, xla::int64 lines, xla::int64 cols) {
```

Comment:
Is the `eye()` above still used?

Reply:
Yep it's still used here. https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor_ops.cpp#L80